test-clients/src/main/java/com/hedera/services/yahcli/config/ConfigManager.java

```diff
@@ -83,6 +83,10 @@ public Map<String, String> asSpecConfig() {
         return specConfig;
     }
 
+    public boolean isALLowListExist(long account) {
+        return targetNet.getAllowList().contains(account);
```

Comment:
I will need to change the logic null is acceptable and supposed to return true as well in order that the process will proceed.
@tinker-michaelj  correct me if I'm wrong

Reply:
I will need to change the false to true that the process will still proceed if it is null because the allow list as I understood from @tinker-michaelj is not mandatory it is an option to have for DevOps

Reply:
As long as it doesn't throw an NPE I'm good :)